tests/test_band_pass_filter.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+import numpy as np
+
+from audiomentations.augmentations.transforms import BandPassFilter
+
+
+class TestBandPassFilter(unittest.TestCase):
+    def test_band_pass_filter(self):
+        samples = np.array(
+            [
+                [[0.75, 0.5, -0.25, -0.125, 0.0], [0.65, 0.5, -0.25, -0.125, 0.0]],
+                [[0.3, 0.5, -0.25, -0.125, 0.0], [0.9, 0.5, -0.25, -0.125, 0.0]],
+                [[0.9, 0.5, -0.25, -1.06, 0.0], [0.9, 0.5, -0.25, -1.12, 0.0]],
+            ],
+            dtype=np.float32,
+        )
+        sample_rate = 16000
+
+        augment = BandPassFilter(min_center_freq=100, 
+                                 max_center_freq=5000,
+                                 min_q=1.0,
+                                 max_q=2.0,
+                                 p=1.0)
+        processed_samples = augment(
+            samples=samples, sample_rate=sample_rate
+        )
+        self.assertEqual(processed_samples.shape, samples.shape)
+        self.assertEqual(processed_samples.dtype, np.float32)
```

Comment:
It could be useful to check that `processed_samples` is not equal to `samples` here

https://stackoverflow.com/questions/38506044/numpy-testing-assert-array-not-equal

Reply:
But it's possible to have `processed_samples` equal to `samples` if signal fits to bandwidth of the filter.

Reply:
Yes, but that can be solved by setting a seed for the random generator in the beginning of the test, so that the same parameters are used every time the test is run.

The intention of this extra assertion would be to check that the transform wasn't accidentally a no-op :)

Reply:
No-op check can be conducted now by checking output plots.